app/forms/location_finder_form.rb

```diff
@@ -30,7 +30,7 @@ def form_sym
   end
 
   def assign_params
-    @file = params[:location_finder_form][:file]
+    @file = params[:location_finder_form]&.fetch(:file, nil)
```

Comment:
Are you expecting the `location_finder_form` to sometimes be `nil` as well?  If not, you won't need the `&`.

Reply:
If the form is submitted with no data, as was previously possible, location_find_form could be nil. However since I added a required flag to the file upload it should no longer be possible for it to be nil but I didn't see the harm in leaving it in.